src/main/java/com/exasol/sql/expression/rendering/ValueExpressionRenderer.java

```diff
@@ -24,8 +28,23 @@ public void visit(final IntegerLiteral literal) {
         append(literal.toString());
     }
 
+    @Override
+    public void visit(final ColumnReference columnReference) {
+        final String tableName = columnReference.getTableName();
+        if ((tableName != null) && !tableName.isEmpty()) {
```

Comment:
Why tableName != null is in the brackets and !tableName.isEmpty() is not? It's somehow inconsistent.

Reply:
`!` is a unary operator while `!=` is a binary operator. Unary operator precedence is always obvious even without extra brackets.